integration/mediation-tests/tests-mediator-2/src/test/java/org/wso2/carbon/esb/mediator/test/iterate/IterateExpressionLessTestCase.java

```diff
@@ -36,29 +36,24 @@
     @BeforeClass(alwaysRun = true)
     public void uploadSynapseConfig() throws Exception {
         super.init();
-
     }
 
     /**
      * This is a negative test under iterator mediator.It will not specify an iterator expression and will try to create
      * a sequence.A sequence cannot be created because  SequenceMediator Sequence named Value cannot be found.
      */
 
-    @Test(groups = { "wso2.esb" }, description = "Testing when there is no iterate expression is specified")
+    @Test(groups = {"wso2.esb"}, description = "Testing when there is no iterate expression is specified")
     public void testIterateExpressionLessSequenceAdding() throws Exception {
-        try {
-            loadESBConfigurationFromClasspath("/artifacts/ESB/mediatorconfig/iterate/iterator_expressionLess.xml");
-            Assert.fail(
-                    "This Configuration can not be saved successfully"); // This will execute when the exception is not thrown as expected
-        } catch (AxisFault message) {
-            assertTrue(message.getMessage().contains(ESBTestConstant.ERROR_ADDING_SEQUENCE),
-                    "Error adding sequence : null not contain in Error Message");
-        }
-    }
 
-    @AfterClass(alwaysRun = true)
-    public void close() throws Exception {
-        super.cleanup();
-    }
+        SimpleHttpClient client = new SimpleHttpClient();
+        Map<String, String> headers = new HashMap<>();
+        headers.put("Accept", "application/json");
+        String endpoint = "https://localhost:9354/management/sequences?sequenceName=invalidIterateMessages";
```

Comment:
It would be better if we move this to a util method so that others also call this Management passing some arguments and get the result.

If we can implement these methods https://github.com/wso2/micro-integrator/blob/master/integration/tests-common/integration-test-utils/src/main/java/org/wso2/esb/integration/common/utils/ESBIntegrationTest.java#L882.L913 , it can be used from everywhere .